z3-sys/src/lib.rs

```diff
@@ -350,7 +350,7 @@ pub enum ParameterKind {
     FuncDecl = generated::Z3_parameter_kind::Z3_PARAMETER_FUNC_DECL as u32,
 }
 
-/// The different kinds of Z3 types (See [`Z3_get_sort_kind`](fn.Z3_get_sort_kind.html)).
+/// The different kinds of Z3 types.
 ///
 /// This corresponds to `Z3_sort_kind` in the C API.
```

Comment:
Can this link to the docs for `Z3_sort_kind` in `z3-sys`? (Are there docs?)

Reply:
To the best of my knowledge, not directly. Indirectly it can point to an absolute URL for a given z3-sys version, but then it becomes tedious to update.

This change is only meant to remove the previously wrong/dead link.

Reply:
Okay. That might be why I didn't do it orginally then. (I think I did that original "This corresponds to ..." stuff.)